include/seqan3/alphabet/nucleotide/nucl16.hpp

```diff
@@ -315,9 +333,140 @@ constexpr nucl16 nucl16::W{internal_type::W};
 constexpr nucl16 nucl16::Y{internal_type::Y};
 constexpr nucl16 nucl16::UNKNOWN{nucl16::N};
 
+} // namespace seqan3
+
+namespace seqan3::detail
+{
+
+//!\brief seqan3::nucl16 is defined as being a nucleotide alphabet.
+template <>
+struct is_nucleotide<nucl16> : public std::true_type
+{};
+
+} // namespace seqan3::detail
+
 #ifndef NDEBUG
-static_assert(alphabet_concept<nucl16>);
-static_assert(detail::internal_alphabet_concept<nucl16>);
+static_assert(seqan3::alphabet_concept<seqan3::nucl16>);
+static_assert(seqan3::nucleotide_concept<seqan3::nucl16>);
 #endif
 
+// ------------------------------------------------------------------
+// containers
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+    /*!\name Alphabet aliases
+     * \{
+     * \brief Other names (typedefs) for seqan3::nucl16
+     * \relates nucl16
+     */
+    using dna16 = nucl16;
+    using rna16 = nucl16;
+    using dna = nucl16;
+    using rna = nucl16;
```

Comment:
Ah, yes, this is also a design decision that I didn't document in the description of the PR. Since even is SeqAn2 people only used the bitcompression rarely we should use the most generic nucleotide type by default. (for regular containers sizeof(dna4 ) == sizeof(dna16) == 8bit)